scripts/twister

```diff
@@ -291,7 +265,7 @@ Artificially long but functional example:
         help="Load list of tests and platforms to be run from file.")
 
     case_select.add_argument(
-        "-T", "--testcase-root", action="append", default=[],
+        "-T", "--testsuite-root", action="append", default=[],
```

Comment:
> Please consider if changing this option name is reasonable. I interpret, that --testcase-root option indicate on folder which contains testcase.yaml files, not testsuite.yaml files. Especially for the beginners it could be unclear.

As a followup, all testcase.yaml files will be renamed to testsuite.yaml. Did not want to do it in this PR.